README.md

```diff
@@ -14,7 +14,7 @@
 - [Integrating SAML into your Java application](#integrating-saml-into-your-java-application)
   * [Getting Started](#getting-started)
     + [Prerequisites](#prerequisites-1)
-  * [Configuring the web app](#configuring-the-web-app)
+  * [Configuring The Sample](#configuring-the-sample)
   * [Enable Login](#enable-login)
   * [Enable Logout](#enable-logout)
   * [Retrieving User Attributes](#retrieving-user-attributes)
```

Comment:
Also need update following reference,
"Throughout this guide we will refer to Identity Server installation directory as IS_HOME and SampleApp installation directory as APP_HOME."

SampleApp -> sample-app

Also IS_HOME is only referenced in one place. So, don't need to have IS_HOME introduced at the top, rather can move to the section it used.